src/sparseml/pytorch/utils/exporter.py

```diff
@@ -444,6 +452,13 @@ def export_onnx(
         See more on the torch.onnx.export api spec in the PyTorch docs:
         https://pytorch.org/docs/stable/onnx.html
     """
+    if _PARSED_TORCH_VERSION >= version.parse("1.10.0") and opset < 13 and convert_qat:
```

Comment:
might want to think about moving out parts of this function into helpers just to improve flow with the torch version checking